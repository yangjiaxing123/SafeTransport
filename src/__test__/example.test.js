/* eslint-disable no-unused-vars */

import React from 'react';
import renderer from 'react-test-renderer';
import Example from '../components/Example';

describe('Example component renders correctly', () => {
  it('renders correctly', () => {
    const rendered = renderer.create(
      <Example />
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});