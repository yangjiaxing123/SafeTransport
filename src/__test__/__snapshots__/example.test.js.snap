// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage component renders correctly renders correctly 1`] = `
<div>
  Example
</div>
`;
